crispy_forms/templatetags/crispy_forms_field.py

```diff
@@ -130,7 +125,7 @@ def render(self, context):  # noqa: C901
                 attribute = template.Variable(attribute).resolve(context)
 
                 if attribute_name in widget.attrs:
-                    if attribute not in widget.attrs[attribute_name]:
+                    if attribute not in widget.attrs[attribute_name].split():
```

Comment:
Added `test_form_control_size` to cover this change. Before this change `form-control` would not get added if `form-control-lg` was already applied; however we need both. Splitting them into a list ensure we can capture unique items.